docs/embroider-css-modules-temporary/types/docs-app-for-embroider-css-modules-temporary/index.d.ts

```diff
@@ -0,0 +1,34 @@
+import '@glint/environment-ember-loose';
+import '@glint/environment-ember-template-imports';
+import 'ember-source/types';
+import 'ember-source/types/preview';
+
+import { HelperLike } from '@glint/template';
+import type EmbroiderCssModulesTemporaryRegistry from 'embroider-css-modules-temporary/template-registry';
+import type SampleV2AddonRegistry from 'sample-v2-addon/template-registry';
+
+type LocalClassHelper = HelperLike<{
+  Args: {
+    Named: {
+      from?: string;
+    };
+    Positional: [localClassNames: string];
+  };
+  Return: string;
+}>;
```

Comment:
A custom definition for the `{{local-class}}` helper from `ember-css-modules`.

Related: https://github.com/salsify/ember-css-modules/pull/301